README.md

```diff
@@ -4,2511 +4,24 @@ ImageViewer is a web application, which allows a user to view his/her own profil
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-Below you will find some information on how to perform common tasks.<br>
-You can find the most recent version of this guide [here](https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md).
+To Run this project just run the below command
 
-## Table of Contents
+  `yarn start`
```

Comment:
We are using `npm` for our app and not `yarn`, let's keep README as original.

Reply:
will change this